### ISSUE TITLE
Does Stark round up contrast ratios?

### ISSUE THREAD

**Issue by JasonBernert** [stark-contrast/stark-sketch-plugin#52]
Had an interesting discrepancy between the Stark Sketch plugin and our Storybook accessibility plugin. We were checking the contrast ratio for the color value `#a96805`. Stark gave it a passing rating of 4.5:1 where the storybook and the [WebAIM checker](https://webaim.org/resources/contrastchecker/#:~:text=WCAG%202.1%20requires%20a%20contrast,4.5%3A1%20for%20large%20text.) fails the contrast ratio.

### Screenshot from Stark
<img width="430" alt="Screen Shot 2020-09-15 at 2 49 12 PM" src="https://user-images.githubusercontent.com/2956240/93358725-d3cf5680-f80f-11ea-84c7-c2fe9c5aade9.png">

### Screenshot from WebAIM
![image-20200916-152515](https://user-images.githubusercontent.com/2956240/93358729-d5008380-f80f-11ea-8143-09663bf3952f.png)

**Comment by imcatnoone** [comment]
As stated in our message, thank you tons for this ticket. The funny part is that you just got in on the actual cusp of our rectifying that rounding. Confirming this has been fixed in the latest [v2.22 release](https://github.com/stark-contrast/stark-sketch-plugin/releases/tag/v2.22)! 👍
